worlds/ahit/Locations.py

```diff
@@ -0,0 +1,980 @@
+
+    "Murder on the Owl Express (Zero Jumps)": LocData(0, "Murder on the Owl Express",
+                                                      required_hats=[HatType.ICE],
+                                                      dlc_flags=HatDLC.death_wish),
+
+    "Picture Perfect (Zero Jumps)": LocData(0, "Picture Perfect", dlc_flags=HatDLC.death_wish),
+
+    "Train Rush (Zero Jumps)": LocData(0, "Train Rush",
+                                       required_hats=[HatType.ICE],
+                                       hookshot=True,
+                                       dlc_flags=HatDLC.death_wish),
+
+    "Contractual Obligations (Zero Jumps)": LocData(0, "Contractual Obligations",
+                                                    paintings=1,
+                                                    dlc_flags=HatDLC.death_wish),
+
+    "Your Contract has Expired (Zero Jumps)": LocData(0, "Your Contract has Expired",
+                                                      umbrella=True,
+                                                      dlc_flags=HatDLC.death_wish),
+
+    # No ice hat/painting required in Expert
+    "Toilet of Doom (Zero Jumps)": LocData(0, "Toilet of Doom",
+                                           hookshot=True,
+                                           hit_requirement=1,
+                                           required_hats=[HatType.ICE],
+                                           paintings=1,
+                                           dlc_flags=HatDLC.death_wish),
+
+    "Mail Delivery Service (Zero Jumps)": LocData(0, "Mail Delivery Service",
+                                                  required_hats=[HatType.SPRINT],
+                                                  dlc_flags=HatDLC.death_wish),
+
+    "Time Rift - Alpine Skyline (Zero Jumps)": LocData(0, "Time Rift - Alpine Skyline",
+                                                       required_hats=[HatType.ICE],
+                                                       hookshot=True,
+                                                       dlc_flags=HatDLC.death_wish),
+
+    "Time Rift - The Lab (Zero Jumps)": LocData(0, "Time Rift - The Lab",
+                                                required_hats=[HatType.ICE],
+                                                dlc_flags=HatDLC.death_wish),
+
+    "Yellow Overpass Station (Zero Jumps)": LocData(0, "Yellow Overpass Station",
+                                                    required_hats=[HatType.ICE],
+                                                    hookshot=True,
+                                                    dlc_flags=HatDLC.dlc2_dw),
+
+    "Green Clean Station (Zero Jumps)": LocData(0, "Green Clean Station",
+                                                required_hats=[HatType.ICE],
+                                                dlc_flags=HatDLC.dlc2_dw),
+}
+
+# noinspection PyDictDuplicateKeys
+snatcher_coins = {
```

Comment:
The thing is, a lot of the Snatcher Coins are only accessible from a specific handful of acts in each chapter, so instead of adding specific rules for each coin, I just put duplicates of them in each act region that they are accessible in.

Reply:
Separately from the dict keys issue, I'm not sure if AP allows "the same" location to be in multiple regions without causing other problems. I feel like this is usually accomplished by making a special "Mafia Town Snatcher Coin" region just for that one location, and then giving all of the "normal" regions an entrance to "Mafia Town Snatcher Coin".

Unrelated: The "Add single comment" button is back! I have no idea what's making it appear and disappear, but I'm not crazy!

![image](https://github.com/ArchipelagoMW/Archipelago/assets/5285357/430895ed-9cc8-4c4e-932e-4f1deb0d3637)

Reply:
I still don't see it, but this has been fixed